src/native/libs/System.Security.Cryptography.Native.Apple/pal_ssl.m

```diff
@@ -0,0 +1,429 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
```

Comment:
This file looks C with some AppleClang extensions, rather than objective C

Reply:
This is `Objective-C` e.g. `.m` vs `.c` file type.